src/client/scripts/components/Navbar1.js

```diff
@@ -14,40 +13,70 @@ export class Navbar1 {
     async render() {
         const elm = document.createElement('nav');
         elm.classList.add('navbar1');
+        
```

Comment:
imma be real w you it looks kinda messed up
![image](https://github.com/ashleybhandari/team-friendship/assets/99090595/480f05e5-451c-408f-9a06-146e0616ff1b)